source/bin/restRoot.cxx

```diff
@@ -103,6 +103,36 @@ int main(int argc, char* argv[]) {
                 runTmp->GetEntry(0);
             }
 
+            // command line AnalysisTree object
```

Comment:
This addon was not mentioned in the PR description

Reply:
OK, I updated the description.